content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,237 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
    Epson and Star receipt printers and Zebra label printers do not need a driver to work. Make sure
    that no driver is selected for those printers.
 
+Epson special case
+~~~~~~~~~~~~~~~~~~
+
+When printing a receipt with an Epson printer, the command that is used is called `GS v 0`. However,
+there are some Epson printers models that do not support this command, they include the following
+models:
+
+- TM-U220
+- TM-U230
+- TM-P60
+- TMP-P60II
+
+If the :abbr:`POS (Point of Sale)` system is using one of these printer models, a particular
+process must be undertaken in order to use another command; `ESC *`.
+
+Process to force ESC * command
+******************************
+
+The first step is to check whether the printer is incompatible with `GS v 0` command.
+
+.. seealso::
+   See this `documentation on Epison's site
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=94>`_ for compatible
+   printers.
+
+Should the printer not be compatible with the `GS v 0` command ensure that it is
+compatible with the `ESC *` command.
+
+.. seealso::
+   See the `Epson documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ for compatible
+   printers.
+
+If the printer is not compatible with the `ESC *` command then the following process is
+not possible. Should the printer be compatible to use the `ESC *` command to print, follow this
+process to configure the printer with the :abbr:`IoT (Internet of Things)` box.
+
+To manipulate the :abbr:`IoT (Internet of Things)` box to use the `ESC *` command to print, go to
+the :abbr:`IoT (Internet of Things)` box homepage. Access the :abbr:`IoT (Internet of Things)` box
+homepage by entering the :abbr:`IP (Internet Protocol)` address of the :abbr:`IoT (Internet of
+Things)` box into a browser window from a computer on the same network as the :abbr:`IoT (Internet
+of Things)` box (preferably by ethernet connection).
+
+.. note::
+   The :abbr:`IP (Internet Protocol)` address can be accessed by the router admin console that the
+   :abbr:`IoT (Internet of Things)` box is connected to, or by connecting a working receipt printer
+   to the :abbr:`IoT (Internet of Things)` box. A receipt will print out with the :abbr:`IoT
+   (Internet of Things)` box's :abbr:`IP (Internet Protocol)` address on it.
```

Comment:
@tiku-odoo  
+ it might be even more confusing as, if I am not mistaken, if the printer is in the network it will print it's IP address (and not the one of the IoT box). Generally they will be connected in USB, but it might not always be the case.

++ in the case of the TMU220 it is a thermal printer so it will not print any ticket (I think)

Reply:
I've removed this section from the doc. This change won't be necessary. Thank you! 

Tim